basicsr/archs/vgg_arch.py

```diff
@@ -101,11 +101,11 @@ def __init__(self,
                 max_idx = idx
 
         if os.path.exists(VGG_PRETRAIN_PATH):
-            vgg_net = getattr(vgg, vgg_type)(pretrained=False)
+            vgg_net = getattr(vgg, vgg_type)(weights=None)
             state_dict = torch.load(VGG_PRETRAIN_PATH, map_location=lambda storage, loc: storage)
             vgg_net.load_state_dict(state_dict)
         else:
-            vgg_net = getattr(vgg, vgg_type)(pretrained=True)
+            vgg_net = getattr(vgg, vgg_type)(weights=vgg.VGG19_Weights.DEFAULT)
 
         features = vgg_net.features[:max_idx + 1]
```

Comment:
As this change is incompatible with the previous pytorch version.
So it's better to keep the original version, otherwise, it will raise errors for the previous pytorch version.